### PR TITLE
add AddButton

### PR DESCRIPTION
**Pull request by j-toscani**
Build an AddButton to be placed on Home-Screen.

### CODE REVIEW

**Comment by marwinburesch** [comment]
looking good! Thanks a bunch

File: src/components/AddButton.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import styled from "styled-components";
+import Add from "../icons/Add";
+// import PropTypes from "prop-types";
```
Comment (lmachens) [inline src/components/AddButton.js]: Please remove commented out code next time :)

File: src/components/AddButton.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  margin: auto;
+`;
+
+const AddIcon = styled(Add)`
+  height: 24px;
+  width: 24px;
+  margin: auto;
+`;
+
+export default function AddButton() {
```
Comment (lmachens) [inline src/components/AddButton.js]: We need to forward `onClick` later to listen to click events of the new button.

File: src/stories/AddButton.stories.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import AddButton from "../components/AddButton";
+
+export default {
+  title: "AddButton"
+};
+
+export const SubmitNewRestaurant = () => <AddButton></AddButton>;
```
Comment (lmachens) [inline src/stories/AddButton.stories.js]: The story has no connection to restaurants. That's why I wouldn't name the story `SubmitNewRestaurant`. Instead, Default would be a good name

**Comment by lmachens** [comment]
Nice, good job!

**Pull request merged**